service/templates/terms_and_conditions.html

```diff
@@ -0,0 +1,167 @@
+{% extends "layout.html" %}
+{% block body %}
+
+  <a class="back-link" href="#" onclick="history.go(-1); return false;">Back</a>
```

Comment:
Same old, same old...if javascript is disabled, the back link won't do anything and the user is stuck on this page unless they use the browser back button.
Is this accessible?